Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -40,7 +40,8 @@ class MainFrameMenu extends Atomic.ScriptObject {
         MenuItemSources.createMenuItemSource("menu tools", toolsItems);
         MenuItemSources.createMenuItemSource("menu developer", developerItems);
         MenuItemSources.createMenuItemSource("menu help", helpItems);
-
+        this.goScreenshot = 0;
+        this.subscribeToEvent(Atomic.UpdateEvent((ev) => this.handleScreenshot(ev)));
```

Comment:
I can see your point about the Update handlers, though it could take a while to see a performance impact in the Editor. Another option that is available is the TBmessage, which has an ability to post a message after a delay.  This would work for some of the recent Editor features (screenshot, statusbar), but would be Editor + UI specific, which isn't all bad.